scripts/languages.ts

```diff
@@ -27,6 +27,14 @@ const EXTRA_SH_LANGUAGES: SupportLanguage[] = [
     filenames: ['.env.*'],
     vscodeLanguageIds: ['dotenv'],
   },
+  {
+    name: 'nvmrc',
+    since: '0.14.0',
+    parsers: ['sh'],
+    extensions: ['.nvmrc'],
+    filenames: ['.nvmrc'],
+    vscodeLanguageIds: ['ignore'],
```

Comment:
Is this correct?

Reply:
By default VS Code uses plaintext for them. There isn't a built-in language. So I wasn't sure what to put here.

Reply:
Can we just omit `vscodeLanguageIds`?

Reply:
Works for me!